pkgs/development/python-modules/spyder/default.nix

```diff
@@ -41,24 +37,21 @@
 , textdistance
 , three-merge
 , watchdog
+, yapf
 }:
 
 buildPythonPackage rec {
   pname = "spyder";
-  version = "5.4.2";
+  version = "5.4.5";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-L8zgT7M7N+P5/9FQgf6ab7afUQXC1afzwUjAp6yKxC8=";
+    hash = "sha256-/9p/8avjy1c3Dwos9Byx03kfVrRofVQus+Ae5beFnmo=";
   };
 
-  patches = [
-    ./dont-clear-pythonpath.patch
```

Comment:
Was this merged upstream/not needed anymore?

Reply:
Actually it is still relevant, my mistake.
I just added it back.